docs/typeclasses/decidable.md

```diff
@@ -0,0 +1,61 @@
+# Decidable
+
+The `Decidable` type class is an extension of [`ContravariantMonoidal`](contravariantmonoidal.html) that allows the
+definition of a `sum` function, allowing us to make choices in consuming our input.
+
+It is the contravariant analogue of the [`Alternative`](alternative.html) type class.
+
+```scala mdoc:silent
+import cats.ContravariantMonoidal
+import cats.data.INothing
+
+trait Decidable[F[_]] extends ContravariantMonoidal[F] {
```

Comment:
I think a good observation might be if you have a `Decidable[F]` then in scala 3 you can derive instances for `F[_]` for enums.

It might even be cool to have some scala 3 specific code that does this.

Reply:
That's a good point, I can look into adding some. If not in this diff then maybe in a follow up, at least.

Reply:
Let's do it separately, this PR has been open for way too long anyway ;)